cmake/find_modules/python/FindPython.cmake

```diff
@@ -1,181 +1,55 @@
-# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
Is this change supposed to be here?  It seems to be a mix-up with the other pull request?

Reply:
yes, this is bad I am not sure how to fix it...